restorer_test.go

```diff
@@ -496,95 +496,6 @@ func testRestorerBuilder(buildpackAPI, platformAPI string) func(t *testing.T, wh
 					metadata := h.MustReadFile(t, filepath.Join("testdata", "analyzer", "app_metadata.json"))
 					h.AssertNil(t, json.Unmarshal(metadata, &restorer.LayersMetadata))
 				})
-
```

Comment:
Can you please explain why it's ok to delete the tests below?
Maybe I missed it but I didn't see that you moved them to a different file.

Reply:
See https://github.com/buildpacks/lifecycle/pull/605#issuecomment-838923343 - but basically these were already brought over in the layer metadata analyzer tests when it was written, but left here initially as well.

Reply:
ohh, ok. I'll take a look. Thanks!